karma.conf.js

```diff
@@ -0,0 +1,101 @@
+// Karma configuration
+// Generated on Tue Oct 03 2017 21:21:19 GMT-0500 (CDT)
+
+module.exports = function (config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['mocha'],
+
+    // list of files / patterns to load in the browser
+    files: [
+      'test/es5.js',
```

Comment:
fwiw, I just copy-pasted this file because it was easier: https://github.com/inexorabletash/polyfill/blob/master/es5.js

I'm assuming we can pull it in via npm